app/api/chat/route.ts

```diff
@@ -32,5 +32,16 @@ export async function GET(request: NextRequest) {
     hasData = hasNext;
   }
 
-  return NextResponse.json({ data: messages });
+  const formattedMessages = messages
+    .map(message => {
+      return {
+        id: message.id,
+        message: message.message,
+        senderId: message.senderId,
+        createdAt: message.createdAt,
+      };
+    })
```

Comment:
제가 알기로는 화살표 함수` () => {}` 이렇게 사용하면` return` 부분은 생략해도 되는걸로 알고 있었는데 틀리면 말씀해주세요~!
```suggestion
    .map(message => {
        id: message.id,
        message: message.message,
        senderId: message.senderId,
        createdAt: message.createdAt,
    })
```

Reply:
저도 `{}`을 사용할 경우 `return`문을 생략해도 되는 것으로 알고 있는데, 오류가 발생합니다!